src/services/shims.ts

```diff
@@ -779,6 +798,35 @@ module ts {
                 });
         }
 
+        public getTSConfigFileInfo(fileName: string, sourceTextSnapshot: IScriptSnapshot): string {
+            return this.forwardJSONCall(
+                "getTSConfigFileInfo('" + fileName + "')",
+                () => {
+                    let text = sourceTextSnapshot.getText(0, sourceTextSnapshot.getLength());
+
+                    try {
+                        var json = /\S/.test(text) ? JSON.parse(text) : {};
+                    }
+                    catch (e) {
+                        return {
+                            options: {},
+                            files: [],
+                            errors: realizeDiagnostic(createCompilerDiagnostic(Diagnostics.Unable_to_open_file_0, fileName), '\r\n')

```

Comment:
you need a more appropriate error.  More like "Failed to parse <file name>.  Parse error was 'e.message'"